exercises/minesweeper/minesweeper_test.py

```diff
@@ -10,114 +10,158 @@
 
 from minesweeper import board
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
+
 
 class MinesweeperTest(unittest.TestCase):
-    def test_board1(self):
-        inp = ["+------+",
-               "| *  * |",
-               "|  *   |",
-               "|    * |",
-               "|   * *|",
-               "| *  * |",
-               "|      |",
-               "+------+"]
-        out = ["+------+",
-               "|1*22*1|",
-               "|12*322|",
-               "| 123*2|",
-               "|112*4*|",
-               "|1*22*2|",
-               "|111111|",
-               "+------+"]
+    # Tests in canonical data incompatible with board class
+    # def test_no_rows(self):
+    #     self.assertEqual(board([]), [])
+
+    # def test_no_columns(self):
+    #     self.assertEqual(board([""]), [""])
```

Comment:
Can you please give a more detailed explanation for why these tests are disabled? I don't like setting a precedent for skipping/disabling canonical tests.

Reply:
Originally this was because the implementation of the board class uses the verify_board function to reject boards without the border or no rows/columns but the canonical data looks like it expects a return of the board as is for both of these cases.

If we'd be okay with it, I can reenable both of these cases but convert both into checks that the board class raises a ValueError in response to the board type instead.

Reply:
I would prefer that you re-enable the disabled test cases, but instead of converting the tests, can we simply do away with the `verify_board` method? There is no requirement in either the [canonical data](https://github.com/exercism/problem-specifications/blob/7735764b0882360563772aa66a2b1fd77c5bb568/exercises/minesweeper/canonical-data.json) or the [description](https://github.com/exercism/problem-specifications/blob/7735764b0882360563772aa66a2b1fd77c5bb568/exercises/minesweeper/description.md) that the solution needs to validate the board; it is merely meant to convert it.

Reply:
We could remove the verify_board method from the example implementation, but I think it would necessitate removing a few of the additional test cases as well (the final 3 that test malformed boards). 

I think it may be desirable to have a verify_board method though. If there's no restriction on the input being a well formed board especially cases with rows/columns of differing sizes it may be hard to decide what's truly adjacent to a mine thus complicating generating output.

It also seems like the no row/no column board  with no border might not be valid given that all  the boards in the problem description and old test set always include a drawn border.

Reply:
Personally, these don't look like valid boards to me, given the borders present in all other examples, so I would expect these to raise `ValueError`s. Ultimately, I don't mind whether these tests are modified or removed - I don't think that they add anything to what we already have.

@cmccandless, I like the inclusion of board verification, and I don't think that it makes sense to remove it in order to better match the canonical data for two trivial cases.